Nighthold/Aluriel.lua

```diff
@@ -85,6 +86,7 @@ function mod:OnBossEnable()
 	self:Log("SPELL_AURA_APPLIED", "PreMarkOfFrostApplied", 212531)
 	self:Log("SPELL_AURA_APPLIED", "MarkOfFrostApplied", 212587)
 	self:Log("SPELL_AURA_APPLIED", "Frostbitten", 212647)
+	self:Log("SPELL_AURA_REMOVED", "FrostbittenRemoved", 212647)

```

Comment:
Should be after in the line after
`self:Log("SPELL_AURA_APPLIED_DOSE", "Frostbitten", 212647)`